chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,6 +150,12 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
+            # we want disks that are multipath but not if they are holders
+            # of a multipath
+            if d.multipath?
```

Comment:
Oh, I suggested this one for no particular reason. So please pick one style that you find best.  Your first proposal looks good style-wise to me, but must be 
```ruby
!(d.claimed? || d.held_by_multipath?) &&
(d.fixed || d.multipath?)
```
otherwise a claimed multipath device could be included in the result.